src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -201,6 +201,7 @@ class MulticopterAttitudeControl
 		param_t roll_tc;
 		param_t pitch_tc;
 		param_t vtol_opt_recovery_enabled;
+		param_t vtol_wv_yaw_rate_d;
```

Comment:
@sanderux how about changing the name to vtol_wv_yawrate__max? It's not really damping anyway.